### PR TITLE
Move from Community Clusters to Cirrus Cloud Clusters wording

### PR DESCRIPTION
**Pull request by fkorotkov**
"community" word might be a bit misleading. One can think of community clusters being managed not by us.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ task:
 It means that Cirrus CI haven't heard from the agent for quite some time. In 99.999% of the cases 
 it happens because of two reasons:
 
-1. Your task was executing on [Community Cluster](guide/supported-computing-services.md#community-cluster). Community Cluster 
+1. Your task was executing on a [Cirrus Cloud Cluster](guide/supported-computing-services.md#cirrus-cloud-clusters).Cirrus Cloud Cluster
```
Comment (edigaryev) [inline docs/faq.md:91]:
```suggestion
1. Your task was executing on a [Cirrus Cloud Cluster](guide/supported-computing-services.md#cirrus-cloud-clusters). Cirrus Cloud Cluster 
```

File: docs/guide/writing-tasks.md [hunk 2]
```diff
@@ -1232,8 +1232,8 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
         ```
 
 ??? warning
-    **Note** that `additional_containers` can be used only with the [Community Cluster](writing-tasks.md)
-    , a [GKE](supported-computing-services.md#kubernetes-engine) cluster or a [EKS](supported-computing-services.md#eks) cluster.
+    **Note** that `additional_containers` can be used only with the [Linux Clusters](writing-tasks.md),
```
Comment (edigaryev) [inline docs/guide/writing-tasks.md:1235]:
```suggestion
    **Note** that `additional_containers` can be used only with the [Linux Clusters](linux.md),
```

File: docs/guide/writing-tasks.md [hunk 3]
```diff
@@ -371,7 +371,7 @@ Note that `pip` cache won't be uploaded in this example: using `upload_caches` d
 
 An `artifacts` instruction allows to store files and expose them in the UI for downloading later. An `artifacts` instruction
 can be named the same way as `script` instruction and has only one required `path` field which accepts a [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming))
-of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G for a community task and with no limit on your own infrastructure.
+of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G for a free task and with no limit on your own infrastructure.
```
Comment (edigaryev) [inline docs/guide/writing-tasks.md:374]:
```suggestion
of files relative to `$CIRRUS_WORKING_DIR` to store. Right now only storing files under [`$CIRRUS_WORKING_DIR` folder](#environment-variables) as artifacts is supported with a total size limit of 1G on Cirrus Cloud Clusters task and with no limit on your own infrastructure.
```

Comment (fkorotkov) [reply docs/guide/writing-tasks.md:374]: There is 5Gb limit for one using compute credits. 🤔

**Review by edigaryev — APPROVED** [review]

**Pull request merged**